packages/webpack.base.config.js

```diff
@@ -56,6 +56,7 @@ module.exports = (env = {}, argv = {}) => {
                  * This is no longer the case. Verify if you need this module and configure a polyfill for it.
                  */
                 timers: false,
+                net: require.resolve('net-browserify'),
```

Comment:
what's this for @leigaol

Reply:
This is a polyfill to resolve the npm run package failure because `net` is not compatible with the web build target. We don't need this poly fill if we don't build target for browser.

Reply:
isee, thanks for the context